src/elements/content-sidebar/activity-feed/task-form/TaskForm.js

```diff
@@ -305,16 +307,18 @@ class TaskForm extends React.Component<Props, State> {
         });
 
         const submitButtonMessage = isCreateEditMode
-            ? messages.tasksAddTaskFormSubmitLabel
-            : messages.tasksEditTaskFormSubmitLabel;
+            ? taskFormMessages.tasksAddTaskFormSubmitLabel
+            : taskFormMessages.tasksEditTaskFormSubmitLabel;
 
-        const taskErrorMessage = isCreateEditMode ? messages.taskCreateErrorMessage : messages.taskUpdateErrorMessage;
+        const taskErrorMessage = isCreateEditMode
+            ? apiMessages.taskCreateErrorMessage
```

Comment:
It seems odd that this one case comes from a different message bundle that's meant to be used by the API.